cfme/configure/access_control.py

```diff
@@ -236,6 +237,11 @@ def remove_tag(self, tag, value):
         form_buttons.save()
         flash.assert_success_message('Tag edits were successfully saved')
 
+    def is_user_present(self):
```

Comment:
We usually call functions of this kind `exists`, see [here](https://github.com/amavinag/cfme_tests/blob/1d04847fb26c668da10f9b5418c31f7992a38718/cfme/configure/access_control.py#L450)